src/components/Planning/PlanningTable/PlanningRow/PlanningHead.tsx

```diff
@@ -75,21 +74,34 @@ const PlanningHead: FC<IPlanningHeadProps> = ({
   /**
    * Adds the currently clicked items id to the search params, expand the row and navigate to the new URL
    */
-  const onExpand = useCallback(() => {
-    handleExpand();
+  const onExpand = useCallback(
+    (event: any) => {
+      dispatchTooltipEvent(event, 'hide', { text: '' });
+      handleExpand();
 
-    const urlSearchParams = new URLSearchParams(search);
-    const searchParams = Object.fromEntries(urlSearchParams.entries());
+      const urlSearchParams = new URLSearchParams(search);
+      const searchParams = Object.fromEntries(urlSearchParams.entries());
 
-    if (urlSearchParam) {
-      Object.assign(searchParams, urlSearchParam);
-    }
+      if (urlSearchParam) {
+        Object.assign(searchParams, urlSearchParam);
+      }
 
-    navigate({
-      pathname: `/${mode}`,
-      search: `${createSearchParams(searchParams)}`,
-    });
-  }, [handleExpand, mode, navigate, search, urlSearchParam]);
+      navigate({
+        pathname: `/${mode}`,
+        search: `${createSearchParams(searchParams)}`,
+      });
+    },
+    [handleExpand, mode, navigate, search, urlSearchParam],
+  );
+
+  const showTooltip = useCallback((event: ReactMouseEvent<HTMLSpanElement>) => {
+    const text =
```

Comment:
Could this be a bit more readable like this?

    const targetElement = event.target as HTMLElement;
    const text = targetElement.textContent || targetElement.innerText;

Reply:
Refactored as suggested